sdk/rust/src/error.rs

```diff
@@ -71,6 +71,7 @@ impl From<hipcheck_common::error::Error> for Error {
 		use hipcheck_common::error::Error::*;
 		match value {
 			UnspecifiedQueryState => Error::UnspecifiedQueryState,
+			UnexpectedRequestInProgress => Error::UnexpectedReplyInProgress,
```

Comment:
Should add another variant to `Error` instead of re-using `UnexpectedReplyInProgress`